docs/self-managed/platform-deployment/amazon-eks.md

```diff
@@ -1,13 +1,14 @@
 ---
 id: amazon-eks
 title: "Amazon EKS"
+description: "Amazon EKS is a managed
```

Comment:
I'm not sure about the convention we use for the description, but I believe it should be something around C8SM and EKS like:

```
description: "Deploy Camunda Platform 8 Self-Managed on Amazon EKS"
```

Reply:
Agreed here. Updated in latest commit.